### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by xtremexp** [xtremexp/UT4X-Converter#45] [News]
This is project is no longer being active and **only blocking bugs** might be fixed. So no new features are expected.

You might report issues here: https://github.com/xtremexp/UT4X-Converter/issues

**Comment by kevincrans** [comment]
Hey winter, I didn't knew it already come so far, too bad it's no longer being active.
I don't know where to post this.
I kind of get it because UT4 and UE4 are also done for and developers like u have an actual life to care about.
I worry about the future of Unreal Tournament and the old engine.

But for now, I can only say, I wish u the best!

**Comment by xtremexp** [comment]
Working back a bit on UT Converter adding Unreal (Tournament) -> UT3 conversion support since Epic Games will only keep UT3 as the latest available UT.
U1/UT99 -> UT3 conversion is now done
UT2003/4 -> UT3 may come in some near future

**Reply by xtremexp** [reply]
Update: since i'm busy with much more important stuff now, project is now currently in 'passive mode'.
Only blocking bugs might be fixed from now and no new feature is expected. (like UE5 support or UT2003/4->UT3 terrain conversion).

**Comment by vrpixel** [comment]
Thanks for your hard work!

**Comment by Bhartman102890** [comment]
Hey Thank you so much, but I REALLY need help converting UDK and importing the t3d into Unreal Engine (4.27) I need a converter. 
I exported a t3d file from UDK, but I have no solutions on converting them for unreal engine.
I made a game in college in 2013. I used UDK 2009.11 
PLEASE HELP! I can even give you the .exe udk file i have PLEASE help 
bhartman102890@gmail.com
Benjamin Hartman
Thankyou
![image](https://github.com/user-attachments/assets/65fcd93e-ffef-49ab-881a-e7f2cbeb0178)

**Reply by kevincrans** [reply]
UT4X converter does supports T3D files and I think it also supports converting from UE3, @xtremexp just forgot to include it in the readme I think.
Else I recommend using the old version https://github.com/RattleSN4K3/UT4Converter , importing the t3d directly (like fbx) might fail.
